core/modules/user/user.pages.inc

```diff
@@ -39,9 +39,13 @@ function user_pass_page() {
  * @see user_pass_validate()
  * @see user_pass_submit()
  */
-function user_pass() {
+function user_pass($form, &$form_state) {
   global $user;
 
+  $user_is_logged_in = user_is_logged_in();
+  if (!$user_is_logged_in) {
+    $form['#theme'] = 'user_login_form';
```

Comment:
In this and other places, I think perhaps using #theme_wrappers might be a better tool, if we're just wrapping this in additional elements. Something like this:

```suggestion
  if ($user_login_page_settings === 'simplified') {
    $form['#theme_wrappers'] = array('user_simplified_page');
  }
```

Reply:
Done